Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionDataRecordInternal.h

```diff
@@ -44,12 +44,17 @@ template<> struct WrapperTraits<WebExtensionDataRecord> {
 
 @property (nonatomic, readonly) WebKit::WebExtensionDataRecord& _webExtensionDataRecord;
 
+@property (readwrite, setter=_setLocalStorageError:) NSError *localStorageError;
+@property (readwrite, setter=_setSessionStorageError:) NSError *sessionStorageError;
+@property (readwrite, setter=_setSyncStorageError:) NSError *syncStorageError;
```

Comment:
These should not need private setters. (See `WebExtensionDataRecord::setError`.)